frontend/src/metabase/query_builder/components/DimensionList.jsx

```diff
@@ -95,17 +104,16 @@ export default class DimensionList extends Component {
     const sectionDimension = dimension
       ? dimension
       : _.find(
-          this._getDimensions(),
-          d => d.field() === item.dimension.field(),
+          this.getDimensions(),
+          dimensionToFind => dimensionToFind.field() === item.dimension.field(),
```

Comment:
Similar to my previous comment, what do you think about renaming this to `dimension` to make it more concise? Would it be more confusing?

Reply:
Far from a strong opinion about this one, happy to update to your suggestion.
Done, in fact.